calendar/calendar.widget/index.coffee

```diff
@@ -64,12 +68,23 @@ updateHeader: (rows, table) ->
     tableRow.append "<td>#{dayOfWeek}</td>"
 
 updateBody: (rows, table) ->
+  #Set to 1 to enable previous and next month dates, 0 to disable
+  PrevAndNext = 1
```

Comment:
I tried putting it at the top, but it gives me the error “no command given”.
> On Aug 4, 2015, at 4:06 AM, Felix <notifications@github.com> wrote:
> 
> In calendar/calendar.widget/index.coffee <https://github.com/felixhageloh/uebersicht-widgets/pull/173#discussion_r36165663>:
> 
> > @@ -64,12 +68,23 @@ updateHeader: (rows, table) ->
> >      tableRow.append "<td>#{dayOfWeek}</td>"
> >  
> >  updateBody: (rows, table) ->
> > +  #Set to 1 to enable previous and next month dates, 0 to disable
> > +  PrevAndNext = 1
> Probably good to move this to the top with all the other configuration option. JS also has true booleans so you can use 'true' and 'false' instead of 0 and 1.
> 
> Finally, it is also a convention to have variable names starting in lowercase (not a big issue of course)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/felixhageloh/uebersicht-widgets/pull/173/files#r36165663>.
>